packages/istanbul-lib-source-maps/package.json

```diff
@@ -7,9 +7,7 @@
   "scripts": {
     "fast": "mocha test/",
     "pretest": "jshint index.js lib/ test/",
-    "test": "istanbul cover --include-all-sources --print=both _mocha --  test/",
-    "xposttest": "istanbul check-coverage --statements 95 --branches 80",
-    "release": "standard-version"
+    "test": "_mocha --  test/"
```

Comment:
this is an Istanbul thing, we could switch to `mocha`; it might be worthwhile to do so.

`_mocha` doesn't spawn a subprocess `mocha` does.